test/util/genesis/document.go

```diff
@@ -50,10 +51,14 @@ func Document(
 
 	authGenState := authtypes.DefaultGenesisState()
 	bankGenState := banktypes.DefaultGenesisState()
+	stakingGenState := stakingtypes.DefaultGenesisState()
+
 	authGenState.Accounts = append(authGenState.Accounts, sdkAccounts...)
 	bankGenState.Balances = append(bankGenState.Balances, genBals...)
 	bankGenState.Balances = banktypes.SanitizeGenesisBalances(bankGenState.Balances)
 
+	stakingGenState.Params.BondDenom = appconsts.BondDenom
```

Comment:
TODO: this shouldn't need to happen due to defaults being set 🤔 need to look into this a bit more